RecFCCeeCalorimeter/src/components/CellPositionsHCalPhiThetaSegTool.h

```diff
@@ -0,0 +1,72 @@
+#ifndef RECCALORIMETER_CellPositionsHCalPhiThetaSegTool_H
+#define RECCALORIMETER_CellPositionsHCalPhiThetaSegTool_H
+
+// Gaudi
+#include "GaudiAlg/GaudiTool.h"
+#include "GaudiKernel/ServiceHandle.h"
+
+// k4geo
+#include "detectorCommon/DetUtils_k4geo.h"
+#include "detectorSegmentations/FCCSWGridPhiTheta_k4geo.h"
+
+// k4FWCore
+#include "k4FWCore/DataHandle.h"
+#include "k4Interface/IGeoSvc.h"
+#include "k4Interface/ICellPositionsTool.h"
+
+// DD4hep
+#include "DD4hep/Readout.h"
+#include "DD4hep/Volumes.h"
+#include "DDSegmentation/Segmentation.h"
+#include <DDRec/DetectorData.h>
+
+// ROOT
+#include "TGeoManager.h"
+
+class IGeoSvc;
+namespace DD4hep {
+namespace DDSegmentation {
+class Segmentation;
+}
+}
+
+/** @class CellPositionsHCalPhiThetaSegTool
+ *
+ */
```

Comment:
the documentation could be expanded a bit (adding also some doxygen style documentation for the methods below)

Reply:
done